HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -2128,6 +2133,9 @@ def to_oga(doc)
         elsif @foundation_type == FoundationTypeCrawlspaceConditioned
           crawlspace = XMLHelper.add_element(foundation_type_el, 'Crawlspace')
           XMLHelper.add_element(crawlspace, 'Conditioned', true, :boolean)
+        elsif @foundation_type == FoundationTypeBellyAndWing
+          belly_and_wing = XMLHelper.add_element(foundation_type_el, 'BellyAndWing')
+          XMLHelper.add_element(belly_and_wing, 'SkirtPresent', @skirt_present, :boolean)
```

Comment:
Are you going to make this a required or optional element? If optional, this would become:
```suggestion
          XMLHelper.add_element(belly_and_wing, 'SkirtPresent', @skirt_present, :boolean, @skirt_present_isdefaulted) unless @skirt_present.nil?
```
and then you'd default the value in `hpxml_defaults.rb`.

Reply:
I'm leaving it as required unless you have a good reason I shouldn't. It seems like an important piece of information that shouldn't be defaulted.

Reply:
I don't really have a strong opinion. But some software tools may not collect this information, so you'd be forcing them to choose something (or add an input).